### PR TITLE
Updating preview sample app to support target os 11.1+. Preview SDK n…

### PR DESCRIPTION
**Pull request by jdizoglio**
…ow has 32 bit(arm7, armv7) architecture support removed

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdizoglio has signed the CLA. Thanks for the pull request!

File: BoxPreviewSDKSampleApp/BoxPreviewSDKSampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -432,14 +432,15 @@
...
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.1;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.box.ios.sampleapp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 3.0;
+				VALID_ARCHS = "arm64 arm64e";
```
Comment (jlawton) [inline BoxPreviewSDKSampleApp/BoxPreviewSDKSampleApp.xcodeproj/project.pbxproj]: Are you sure this is needed?

Comment (jdizoglio) [reply BoxPreviewSDKSampleApp/BoxPreviewSDKSampleApp.xcodeproj/project.pbxproj]: I removed arm7 and it seems to Xcode's writes the entry.

Comment (jlawton) [reply BoxPreviewSDKSampleApp/BoxPreviewSDKSampleApp.xcodeproj/project.pbxproj]: What I meant was, do we need to remove armv7, or is that implicit in the deployment target?

**Pull request merged**